package.json

```diff
@@ -1,12 +1,12 @@
 {
   "main": "./integration.js",
   "name": "microsoft-sentinel",
-  "version": "3.0.0-beta",
+  "version": "3.0.1-beta",
```

Comment:
Since we added a new option which counts as a feature I would probably bump to 3.1.0